internal/controller/streams.go

```diff
@@ -236,6 +272,26 @@ func connectAndStreamCiliumNetworkFlows(logger *zap.SugaredLogger, sm *streamMan
 	return nil
 }
 
+// connectAndStreamFalcoNetworkFlows creates networkFlowsStream client and begins the streaming of network flows.
+func connectAndStreamFalcoNetworkFlows(logger *zap.SugaredLogger, sm *streamManager) error {
+	falcoCtx, falcoCancel := context.WithCancel(context.Background())
+	defer falcoCancel()
+	sendFalcoNetworkFlows, err := sm.streamClient.client.SendKubernetesNetworkFlows(falcoCtx)
+	if err != nil {
+		logger.Errorw("Failed to connect to server", "error", err)
+		return err
+	}
+	sm.streamClient.networkFlowsStream = sendFalcoNetworkFlows
```

Comment:
How do you make sure that only one of `connectAndStreamCiliumNetworkFlows` or `connectAndStreamFalcoNetworkFlows` gets called and sets this `networkFlowsStream ` field?

Reply:
my thought process is we will always have falco enabled UNLESS Cilium is used. Then if cilium is enabled that falco server will never be used thus its okay for us to override that networkFlowsStream. Now if we are worried there will be some customer environments that may have both, than Cilium should still take precedence (which is what is happening now). Now if they are both set up by a customer, they will not have the illumio specific rules that our falco events are being filtered on thus those events should never even be sent down stream.